### DISCUSSION TITLE
Has Saleor been abandoned?

### DISCUSSION THREAD

**Discussion by weeping-somnambulist** [saleor/saleor#7594] [General]
The saleor-platform just flat out doesnt work, and there have been issue reports reflecting that going back to **MARCH** without so much as an acknowledgement.  Is this project abandoned??? Sure seems like it...

**Comment by karolkielecki** [comment]
@SwitchedToGitlab Hi there! We haven't forgotten about `saleor-platform`. Currently, we have other priorities, including Saleor 3.0. Our main focus is to deliver the features from the [roadmap](https://github.com/mirumee/saleor/projects/12), such as multichannel and much more. I'll keep you posted regarding `saleor-platform`, and if you have any questions, please let me know. 

Cheers,

**Reply by obaviet** [reply]
> @SwitchedToGitlab Hi there! We haven't forgotten about . Currently, we have other priorities, including Saleor 3.0. Our main focus is to deliver the features from the [roadmap](https://github.com/mirumee/saleor/projects/12), such as multichannel and much more. I'll keep you posted regarding , and if you have any questions, please let me know.`saleor-platform``saleor-platform`
> 
> Cheers,
saleor platform  is still very important especially in the case of this experience

**Comment by mehrdad-shokri** [comment]
hello @SwitchedToGitlab I recently ran saleor-platform. The only issue now sticking around is that storefront can't load graphql from localhost:8000/graphql which seems to me to be a docker network problem that I haven't been able to solve although I tried to put storefront and api in the same network but no luck. Othe than that It seemed ok to me. What specific problem are you dealing with? maybe I can help

**Comment by tyrozz** [comment]
I have used saleor-platform with Saleor 3.0. It works. This is how I do: I run Saleor core and everything related the core with docker-compose. 
I removed storefront and dashboard from docker-compose. I run these as described in their Github pages, npm basically. It is faster this way with my computer. 
And everything works. I am exploring Saleor 3.0 this way.

**Reply by obaviet** [reply]
> I have used saleor-platform with Saleor 3.0. It works. This is how I do: I run Saleor core and everything related the core with docker-compose.
> I removed storefront and dashboard from docker-compose. I run these as described in their Github pages, npm basically. It is faster this way with my computer.
> And everything works. I am exploring Saleor 3.0 this way.

How do you remove  storefront and dashboard from docker-compose.?

**Reply by karolkielecki** [reply]
@obaviet We'll update `saleor-platform` in the next few weeks--so it will be working fine. We're aware that there are some issues with the dashboard and the storefront while using the `saleor-platform` repo.
